root/utility/edit.js

```diff
@@ -45,6 +45,13 @@ const STATUS_NAMES = {
   [EDIT_STATUS_OPEN]:           N_l('Open'),
 };
 
+export function getEditCssClassName(edit: EditT): string {
```

Comment:
I *think* `kebabCase` from root/static/scripts/common/utility/strings.js does the same thing.